go.mod

```
@@ -331,7 +331,7 @@ replace (
 	github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v0.0.0-20210511163231-5b2d2b5f6c34
 	github.com/docker/docker => github.com/moby/moby v24.0.7+incompatible
 
-	// Remove once we get to a version with extproc changes in upstream envoy (currently targeted 1.30)
+	// Remove once we get to envoy 1.30
```

nit for discoverability:
```suggestion
	// ON_LTS_UPDATE: Remove once we get to envoy 1.30
```

---

ah good point -- resolving the thread as I think the suggestion probably adds more confusion than just leaving this as-is.